src/GitTfs.VsCommon/TfsHelper.Common.cs

```diff
@@ -1317,7 +1331,7 @@ public void CreateTfsRootBranch(string projectName, string mainBranch, string gi
                                 });
                 }
 
-                workspace = GetWorkspace(new WorkingFolder(projectPath, gitRepositoryPath));
+                workspace = GetWorkspace("", new WorkingFolder(projectPath, gitRepositoryPath));
```

Comment:
This looks strange. Can't we just set the workspace owner in the options to its default `.` and pass it through accordingly, so we don't have these defaults somewhere deep in the stack?

Reply:
Yes we can, I set this in the options also here 5e986260d531e5b11c246224add5a6bbdcdb33df
It really is better this way, thank you!